lib/kitchen/directions/bake_chapter_key_concepts/v1.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeChapterKeyConcepts
+  class V1
+    renderable
+    def bake(chapter:, metadata_source:)
+      metadata_elements = metadata_source.children_to_keep.copy
+      @chapter_metadata = metadata_elements.paste
```

Comment:
This works and is ok, but you could remove line 8 and have line 7 have the `@` up front and then say `@metadata_elements.paste` in the `.xhtml.erb` file.